.github/workflows/nodejs.yml

```diff
@@ -5,12 +5,15 @@ on:
     branches: [master, develop]
   pull_request:
 
+env:
+  PLATFORM_JAVA_OPTS: "-XX:+UnlockExperimentalVMOptions -XX:+UseZGC -Xlog:gc*:gc.log"
```

Comment:
We are gonna remove that environment variable because it's experimental and might not work properly on different platforms, so just keep it empty or remove it at all.
```
PLATFORM_JAVA_OPTS: ""
```